src/bin/index.ts

```diff
@@ -1,60 +1,61 @@
 #! /usr/bin/env node
 
+import fs from "node:fs";
+import { exit } from "node:process";
 import chalk from "chalk";
 import { program } from "commander";
-import fs from "fs";
 import { glob } from "glob";
-import yaml from 'js-yaml';
-import { exit } from "process";
+import yaml from "js-yaml";
 import { checkTranslations, checkUnusedKeys } from "..";
 import { Context, standardReporter, summaryReporter } from "../errorReporters";
 import { CheckResult, FileInfo, TranslationFile } from "../types";
 import { flattenTranslations } from "../utils/flattenTranslations";
 
+const version = require("../../package.json").version;
+
 program
-  .version("0.3.0")
+  .version(version)
   .option(
     "-l, --locales <locales...>",
     "name of the directory containing the locales to validate"
   )
+  .option("-s, --source <locale>", "the source locale to validate against")
   .option(
-    "-s, --source [source locale]",
-    "the source locale to validate against"
-  )
-  .option(
-    "-f, --format [format type]",
+    "-f, --format <type>",
     "define the specific format: i18next or react-intl"
   )
   .option(
-    "-c, --check [checks]",
-    "define the specific checks you want to run: invalid, missing. By default the check will validate against missing and invalid keys, i.e. --check invalidKeys,missingKeys"
+    "-o, --only <only...>",
```

Comment:
Should we have some deprecation or something not to break existing users with this change?

Reply:
Makes sense, updated with a deprecation notice.